packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -237,7 +243,7 @@ describeCompat("blobs", "FullCompat", (getTestObjectProvider, apis) => {
 			});
 
 			const dataStore = await getContainerEntryPointBackCompat<ITestDataObject>(container);
-			const blobOpP = new Promise<void>((resolve, reject) =>
+			const blobOpP = timeoutPromise((resolve, reject) =>
```

Comment:
I wonder if we can get a lint rule to error out if you use `new Promise` in our test files.  We should always use timeoutPromise I think.

Reply:
Yea, Looks like these tests weren't providing as much value just noise. Better to have less of that.